### DISCUSSION TITLE
Mitigation Techniques Used For Vulnerabilities Found In ImageMagick

### DISCUSSION THREAD

**Discussion by AshutoshDKedar** [ImageMagick/ImageMagick#4891] [Help]
What are the techniques or precautions we need to take to mitigate the below listed vulnerabilities.

https://www.cvedetails.com/vulnerability-list/vendor_id-1749/Imagemagick.html

**Comment by dlemstra** [comment]
You can find a lot of information about that here: https://imagemagick.org/script/security-policy.php

**Comment by urban-warrior** [comment]
ImageMagick recommended practices strongly encourage you to configure a [security policy](https://imagemagick.org/script/security-policy.php) that suits your local environment.

Keep ImageMagick up-to-date. The latest releases have fixes for any security flaws we discovered in the past.
